tracer/src/Datadog.Trace/Agent/SpanEventConverter.cs

```diff
@@ -86,6 +84,7 @@ private static bool IsAllowedType(object value)
             }
         }
 
+        // Removed decimal & ulong from supported types to match both Native SpanEvents RFC and JSON events
```

Comment:
Not a blocker, but this _could_ technically be a behavior change if I follow the flow on this

Previously if someone had a `ActivityEvent` with either a `decimal` or `ulong` value we would write it out previously, now we won't. But this is quite the edge case as someone would have to do this via the `Activity` (and it is also unclear what we should do here if they have native span events).

Reply:
This is a good call-out Steven, but worth doing anyways since it aligns us with OpenTelemetry behavior. Perhaps we can include this breaking behavior in our release notes